nservicebus/messaging/send-a-message.md

```diff
@@ -1,27 +1,43 @@
 ---
-title: Sending a Message
-summary: Describes how to send a message
+title: Sending messages
+summary: Describes how to send messages
 tags: []
 redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-Message sending involves using Send operation that takes an argument of a message instance to be delivered. The details differ slightly between version of NServiceBus, as shown in following snippets.
+NServiceBus allows you to send different types of messages (e.g. Commands, Events, Replies, etc.) to itself or other endpoints. Messages can be sent from outside or inside the message processing pipeline:
```

Comment:
In this context "itself" makes no sense. It refers to "nservicebus" and "other endpoints", how about u just go with "any endpoint"